service/src/main/java/com/ge/predix/acs/privilege/management/dao/GraphGenericRepository.java

```diff
@@ -402,7 +401,7 @@ private void searchAttributesWithScopes(final E entity, final Vertex vertex, fin
                         attributes.addAll(deserializedAttributes);
                         // This enforces the limit on the count of attributes returned from the traversal, instead of
                         // number of vertices traversed. To do the latter will require traversing the graph twice.
-                        checkTraversalLimitOrFail(attributes);
+                        checkTraversalLimitOrFail(entity, attributes);
```

Comment:
ok, I created a separate story for this - https://rally1.rallydev.com/#/59288076735ud/detail/userstory/60661742041?fdp=true